### DISCUSSION TITLE
Welcome to Flutter UX Research Discussions!

### DISCUSSION THREAD

**Discussion by InMatrix** [flutter/uxr#48] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with anyone who's interested in UX research on Flutter & Dart. We hope that you:
  * Participate in the discussions of an ongoing research study (e.g., the [Navigator 2.0 API usability research](https://github.com/flutter/uxr/wiki/Navigator-2.0-API-Usability-Research))
  * Share research and design ideas related to Flutter's developer experience
  * Discuss academic or industry projects relevant to Flutter & Dart's developer experience
  * Engage with other community members

This Discussions board follows [Flutter's code of conduct](https://github.com/flutter/flutter/blob/master/CODE_OF_CONDUCT.md). It is not intended to replace any existing issue trackers for Flutter or Dart.

**Comment by Sunbreak** [comment]
Does `UX research on Dart` means `Developer Experience on Dart`?

Dart native interop needs quite a lot docs, e.g. https://github.com/dart-lang/sdk/issues/37022#issuecomment-828619281

**Reply by InMatrix** [reply]
Yes, "user experience" here refers to the experience of developers who use Flutter and Dart. For your specific feedback, I'd suggest opening an issue to request docs for Dart FFI in the Dart SDK repo or the [Dart website](https://github.com/dart-lang/site-www/issues) repo.

**Comment by lightlessdays** [comment]


**Reply by InMatrix** [reply]
Hello, thank you for checking out the repo, but it's not helpful to post just greetings without actual content. I'll remove your two comments to keep this space focused on UX Research discussions. Thank you for your understanding.
